Cesium.CodeGen.Tests/CodeGenMethodTests.cs

```diff
@@ -348,6 +348,9 @@ public void ImplicitReturnDisallowedNonMain() => DoesNotCompile(@"int foo()
     int unused;
 }", "Function foo has no return statement.");
 
+    [Fact]
+    public Task ReturnInsideInfiniteForTest() => DoTest("int test() { for(;;) { return; } }");
```

Comment:
_(Note)_ Codegen for this method is quite suspicious:
```
System.Int32 <Module>::test()
  IL_0000: nop
  IL_0001: ret
  IL_0002: nop
  IL_0003: br IL_0000
  IL_0008: nop
```
`ret` seems to be the first executed instruction.

Can confirm it was always like this (even before your PR).

I'm making a note to open an issue about that after merge.

Reply:
#633.